crates/libp2p-networking/src/network/behaviours/dht/mod.rs

```diff
@@ -277,7 +277,7 @@ impl<K: SignatureKey + 'static> DHTBehaviour<K> {
                     }
                 },
                 Err(err) => {
-                    error!("GOT ERROR IN KAD QUERY {:?}", err);
```

Comment:
Do we get this a lot? I'd be ok lowering even to info

Reply:
Yeah we get it whenever a lookup actually fails